enterprise/dev/ci/ci_dependencies.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
```suggestion
//+build tools

package tools
```

I'd also just rename this `dev/ci/tools` (notably _not_ in `enterprise`)

Reply:
The `+build` should exclude the package, this is what I used to do in https://github.com/bobheadxi/gobenchdata/blob/master/internal/tools/tools.go , I think it _should_ work

Reply:
Arg, this breaks on go tests involving another `go.mod` (https://buildkite.com/sourcegraph/sourcegraph/builds/129938#4d207189-fdaf-4747-8498-92d1cb51e5a7/87). I'll drop the commits for now.